### PR TITLE
Support volume zones and aeroacoustic output

### PR DESCRIPTION
**Pull request by awkrupka**
More complete volume zone tests were added, aeroacoustic output has been defined based on the JSON Schema.

Also incorporates a temporary fix causing extremely long test running times on Windows caused by log file writes - this issue needs to be addressed separately.

### CODE REVIEW

File: flow360/component/flow360_params/flow360_params.py [hunk 1]
```diff
@@ -580,6 +581,166 @@ def validate_boundary(cls, values):
...
+
+    Example
+    -------
+    >>> aeroacoustics = AeroacousticOutput()
+    """
+
+    animation_frequency: Optional[PositiveInt] = pd.Field(alias="animationFrequency")
+    animation_frequency_offset: Optional[int] = pd.Field(alias="animationFrequencyOffset")
+    patch_type: Optional[str] = pd.Field("solid", const=True)
+    observers: List[Coordinate] = pd.Field(alias="observers")
```
Comment (maciej-flexcompute) [inline flow360/component/flow360_params/flow360_params.py:741]: alias seems to be redundant here

File: flow360/component/flow360_params/flow360_params.py [hunk 2]
```diff
@@ -580,6 +581,166 @@ def validate_boundary(cls, values):
...
+        An instance of the component class AeroacousticOutput.
+
+    Example
+    -------
+    >>> aeroacoustics = AeroacousticOutput()
+    """
+
+    animation_frequency: Optional[PositiveInt] = pd.Field(alias="animationFrequency")
+    animation_frequency_offset: Optional[int] = pd.Field(alias="animationFrequencyOffset")
+    patch_type: Optional[str] = pd.Field("solid", const=True)
```
Comment (maciej-flexcompute) [inline flow360/component/flow360_params/flow360_params.py:740]: it needs alias `patchType`

File: flow360/component/flow360_params/flow360_params.py [hunk 3]
```diff
@@ -580,6 +581,166 @@ def validate_boundary(cls, values):
...
+    -------
+    :class:`AeroacousticOutput`
+        An instance of the component class AeroacousticOutput.
+
+    Example
+    -------
+    >>> aeroacoustics = AeroacousticOutput()
+    """
+
+    animation_frequency: Optional[PositiveInt] = pd.Field(alias="animationFrequency")
```
Comment (maciej-flexcompute) [inline flow360/component/flow360_params/flow360_params.py:738]: animationFrequency can be positive int or -1. I think Union with Literal can work here

File: flow360/component/flow360_params/flow360_params.py [hunk 4]
```diff
@@ -580,6 +581,166 @@ def validate_boundary(cls, values):
...
+        the simulation domain.
+
+    Returns
+    -------
+    :class:`AeroacousticOutput`
+        An instance of the component class AeroacousticOutput.
+
+    Example
+    -------
+    >>> aeroacoustics = AeroacousticOutput()
```
Comment (maciej-flexcompute) [inline flow360/component/flow360_params/flow360_params.py:735]: lets give example that means something, for example `observers=[(0, 0, 0), (0, 0, 1)]`

File: flow360/component/flow360_params/flow360_params.py (hunk 1 above)
Comment (awkrupka) [reply flow360/component/flow360_params/flow360_params.py]: Fixed in a01150b

File: flow360/component/flow360_params/flow360_params.py (hunk 2 above)
Comment (awkrupka) [reply flow360/component/flow360_params/flow360_params.py]: Fixed in a01150b

File: flow360/component/flow360_params/flow360_params.py (hunk 3 above)
Comment (awkrupka) [reply flow360/component/flow360_params/flow360_params.py]: Fixed in a01150b, added additional unit tests for -1 and other negative numbers

File: flow360/component/flow360_params/flow360_params.py (hunk 4 above)
Comment (awkrupka) [reply flow360/component/flow360_params/flow360_params.py]: Fixed in a01150b, also added minimal entries for optional parameters in the documentation

**Review by maciej-flexcompute — APPROVED** [review]

**Review by maciej-flexcompute — APPROVED** [review]

**Pull request merged**